Resources/doc/reference/configuration/proxy-client.rst

```diff
@@ -323,6 +323,41 @@ the `AWS Async documentation_`. It can not be used with the ``client`` option.
 Service identifier of a `AsyncAws\CloudFront\CloudFrontClient` client. More information is available on the
 `AWS Async documentation_`. It can not be used with the ``configuration`` option.
 
+.. _configuration_fastly_proxy_client:
+
+Fastly
+----------
+.. code-block:: yaml
+
+    # config/packages/fos_http_cache.yaml
+    fos_http_cache:
+        proxy_client:
+            fastly:
+                service_identifier: '<my-service-identifier>'
+                authentication_token: '<my-authentication-token>'
+                soft_purge: true
+
+``service_identifier``
+"""""""""""""""""""
+
+**type**: ``string``
+
+Identifier for the Fastly Service you want to purge the cache for.
```

Comment:
```suggestion
Identifier for the Fastly account you want to purge the cache for.
```

Reply:
Service is the correct term.

You can have a fastly account with different services (VCL configuration), if you want to purge the cdn(cache) you have to specifiy the service you want to do that in.

Reply:
okay, if that is fastly terminology, lets keep it.

should "service" be lowercase though?